### ISSUE TITLE
RDS popup not accessible for screen readers

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#104]
User is able to specify additional information about RDS which is not visible in radio list's table. Only first frame of RDS PS is shown in radio list's table. Additional information is available in small popup shown when first RDS PS frame is hovered or focused.

![](https://user-images.githubusercontent.com/1395027/82903070-b294e280-9f60-11ea-9ebb-4fa89fab0b9d.png)

Currently, RDS popup is not accessible for screen reader users. 

#15

**Comment by TomaszGasior** [comment]
Extra info from blind user: https://github.com/TomaszGasior/RadioLista-v3/issues/238#issue-2941235416
